.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+<!--
+Thanks for contributing!
+-->
+
+Some things to make sure you have completed:
+- [ ] Bump Spec version.
+- [ ] Update last modified date.
+- [ ] Update changelog.
+- [ ] Make sure there is generated JSON files from the YAML test files.
+- [ ] Test changes in your language driver.
+
+Please review the [spec process wiki page](https://wiki.corp.mongodb.com/pages/viewpage.action?pageId=80806719) for additional information.
+
+<!--
+Thanks again!
+-->
```

Comment:
I understand the good sentiment here, but I imagine this (and the opening comment) are going to be manually deleted each time someone opens a PR. Since the vast majority of spec PRs are opened by staff, I propose we remove these comments and limit this template to the checklist and info link.

```suggestion
```